projects/dev/src/app/group/group-manage/pending-request/pending-request.component.html

```diff
@@ -98,29 +97,41 @@
       </ng-template>
       <ng-template #summaryTemplate let-table>
         <div style="text-align: left; display: flex; align-items: center;">
-          <p-tableHeaderCheckbox></p-tableHeaderCheckbox>
+          <p-tableHeaderCheckbox [disabled]="!isIdle()"></p-tableHeaderCheckbox>
           <span
             style="
               margin-left: 2.1rem;
               color: #9b9b9b;
-              flex: 1;
               font-size: 1.2rem;
+              cursor: pointer;
             "
+            [ngClass]="{disabled: selection.length === 0 || !isIdle()}"
+            (click)="onSelectAll($event)"
           >
             Select all
           </span>
+          <span style="flex: 1;"></span>
           <div class="req-tools">
-            <span class="accept" (click)="onClickAccept($event)">
+            <span class="accept" [ngClass]="{disabled: selection.length === 0 || !isIdle()}" (click)="onAcceptOrReject(Action.Accept)">
               <span>Accept</span>
-              <i class="fa fa-check"></i>
+              <ng-container *ngIf="isAccepting()">
+                <p-progressSpinner></p-progressSpinner>
+              </ng-container>
+              <ng-container *ngIf="!isAccepting()"><i class="fa fa-check"></i></ng-container>
             </span>
-            <span class="reject" (click)="onClickReject($event)">
+            <span class="reject" [ngClass]="{disabled: selection.length === 0 || !isIdle()}" (click)="onAcceptOrReject(Action.Reject)">
               <span>Reject</span>
-              <i class="fa fa-times"></i>
+              <ng-container *ngIf="isRejecting()">
+                <p-progressSpinner></p-progressSpinner>
+              </ng-container>
+              <ng-container *ngIf="!isRejecting()">
+                <i class="fa fa-times"></i>
+              </ng-container>
             </span>
           </div>
         </div>
       </ng-template>
     </app-grid>
   </div>
 </app-section-paragraph>
+<p-toast></p-toast>
```

Comment:
I have 19 problems reported by VSCode (by "ng"?) on the html. For instance: 
```
{
	"resource": "projects/dev/src/app/group/group-manage/pending-request/pending-request.component.html",
	"owner": "_generated_diagnostic_collection_name_#0",
	"severity": 8,
	"message": "Can't bind to 'pSortableColumn' since it isn't a known property of 'th'.",
	"source": "ng",
	"startLineNumber": 23,
	"startColumn": 15,
	"endLineNumber": 23,
	"endColumn": 44
}
{
	"resource": "projects/dev/src/app/group/group-manage/pending-request/pending-request.component.html",
	"owner": "_generated_diagnostic_collection_name_#0",
	"severity": 8,
	"message": "'p-sortIcon' is not a known element:\n1. If 'p-sortIcon' is an Angular component, then verify that it is part of this module.\n2. If 'p-sortIcon' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.",
	"source": "ng",
	"startLineNumber": 28,
	"startColumn": 17,
	"endLineNumber": 28,
	"endColumn": 49
}
{
	"resource": "projects/dev/src/app/group/group-manage/pending-request/pending-request.component.html",
	"owner": "_generated_diagnostic_collection_name_#0",
	"severity": 8,
	"message": "Can't bind to 'field' since it isn't a known property of 'p-sortIcon'.\n1. If 'p-sortIcon' is an Angular component and it has 'field' input, then verify that it is part of this module.\n2. If 'p-sortIcon' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.\n3. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.",
	"source": "ng",
	"startLineNumber": 28,
	"startColumn": 29,
	"endLineNumber": 28,
	"endColumn": 48
}
...
```

Reply:
How did you get this?

Reply:
I get this directly in VSCode (I have the "Angular Language Service" installed).

Reply:
To be ignored as discussed.